.github/workflows/ci-build.yaml

```diff
@@ -420,8 +420,8 @@ jobs:
           # port 8080 which is not visible in netstat -tulpen, but still there
           # with a HTTP listener. We have API server listening on port 8088
           # instead.
-          make controller repo-server server applicationset-controller
-          make start-e2e-local 2>&1 | sed -r "s/[[:cntrl:]]\[[0-9]{1,3}m//g" > /tmp/e2e-server.log &
+          make cli-local
```

Comment:
Also this to be added in start-e2e-local in makefile because building on my local machine is also too heavy. it hangs

Reply:
Agree. Running `cli-local` as part of `start-e2e-local` and using BIN_MODE by default